src/components/AboutUs/AboutUs.test.js

```diff
@@ -0,0 +1,48 @@
+/* eslint-disable prefer-const */
+import React from "react";
+import { Provider } from "react-redux";
+import { BrowserRouter } from "react-router-dom";
+import renderer from "react-test-renderer";
+import configureStore from "redux-mock-store";
+import AboutUs from "./AboutUs";
+
+const mockStore = configureStore([]);
+
+it("renders when user don`t registered", () => {
+  let store;
+  store = mockStore({
```

Comment:
Create and initialize this variable in one line and with `const`.

Reply:
Done